src/concurrent_files.rs

```diff
@@ -92,6 +92,7 @@ where
         }
         if path.is_dir() {
             for entry in WalkDir::new(path)
+                .sort_by_file_name()
```

Comment:
Why is this change required?

Reply:
We found that file access order in a directory depends on the operating system policy. This line should define a unique deterministic exploration order valid for every operating system.

Reply:
Why do we need a deterministic exploration order?

Reply:
You are right. It seems this line is not strictly necessary for the tests to work. Source files can be read in any order since the json file path is deducted directly from the source file path. Comparisons for a single file will never be affected by this change. I will remove this change. Thank you for pointing this out!

Reply:
No problem! I actually didn't know that it wasn't necessary, it's good I asked :)